src/data/Pager/Pager.tsx

```diff
@@ -60,24 +72,33 @@ export interface PagerProps {
    * A callback function that is called when the page changes.
    */
   onPageChange?: (page: number) => void;
+
+  /**
+   * The type of table data for the pager to display.
+   */
+  displayMode?: 'pages' | 'items' | 'all';
```

Comment:
The `items` name isn't the most intuitive to me - what about `controls`?

- `displayMode: 'all' | 'controls' | 'pages';`

Reply:
the `items` option is controlling the display for the `1-10 of 100 items` part not the arrow buttons. Maybe `label` or `rows` could be a diff name?